.github/SECURITY.md

```diff
@@ -2,15 +2,14 @@
 
 ## Supported Versions
 
-Ansible Language Server does not backport security fixes only applying
-them to the main branch and making the next patch release in the stream.
+Ansible Language Server does not backport security fixes only applying them to
+the main branch and making the next patch release in the stream.
 
 ## Reporting a Vulnerability
 
-We encourage responsible disclosure practices for security
-vulnerabilities. Please read our [policies for reporting bugs][bug
-reports policy] if you want to report a security issue that might affect
-Ansible.
+We encourage responsible disclosure practices for security vulnerabilities.
+Please read our [policies for reporting bugs][bug reports policy] if you want to
+report a security issue that might affect Ansible.
 
 [bug reports policy]:
-https://docs.ansible.com/ansible/devel/community/reporting_bugs_and_features.html#reporting-a-bug
+  https://docs.ansible.com/ansible/devel/community/reporting_bugs_and_features.html#reporting-a-bug
```

Comment:
Why indent this?

Reply:
In fact I do remember a rule in markdownlint about splitting long lines. I am surprised that markdown-lint did not complain about that line because that wrapping rule would have requested the wrapping.

That is happening only because the URL is very long, not always.